### PR TITLE
RHIROS 217 - Fix core_collect errors

### PR DESCRIPTION
**Pull request by apuntamb**

### CODE REVIEW

**Review by kgaikwad — CHANGES_REQUESTED** [review]

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 1]
```diff
@@ -8,20 +8,20 @@
...
-      shell: grep -c '^\s*core_collect\s*=\s*True\s*' /etc/insights-client/insights-client.conf || true
-      register: core_collect
     - name: Ensure core_collect is enabled
       lineinfile:
+        state: present
         dest: /etc/insights-client/insights-client.conf
+        insertafter: "^#core_collect=True"
         line: "core_collect=True"
-      when: core_collect.stdout == '0'
+      register: core_collect
```
[the remark this answers is not in the document]
Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml]: done

**Comment by kgaikwad** [comment]
@apuntamb - could you confirm if all playbooks run successfully in check-mode without an error?

**Comment by apuntamb** [comment]
> @apuntamb - could you confirm if all playbooks run successfully in check-mode without an error?

@kgaikwad Yeah, it works fine for all three playbooks in check mode.

**Comment by kgaikwad** [comment]
Thanks @apuntamb :+1: :ok_hand:

**Pull request merged**